docs/trace/getting-started/README.md

```diff
@@ -50,15 +50,39 @@ What does the above program do?
 
 The program creates an `ActivitySource` which represents an [OpenTelemetry
 Tracer](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#tracer).
+
+```csharp
+private static readonly ActivitySource MyActivitySource = new ActivitySource(
+        "MyCompany.MyProduct.MyLibrary");
+```
+
 The `ActivitySource` instance is used to start an `Activity` which represents an
 [OpenTelemetry
 Span](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#span).
+
+```csharp
```

Comment:
Follow the indentation convention:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/bf79f0d40df151824409fc6e1b38988330e96ec1/docs/trace/customizing-the-sdk/Program.cs#L51-L56